torch_em/data/datasets/light_microscopy/covid_if.py

```diff
@@ -93,10 +93,7 @@ def get_covid_if_dataset(
         sample_range: Id range of samples to load from the training dataset.
         target: The segmentation task. Either 'cells' or 'nuclei'.
         download: Whether to download the data if it is not present.
-        offsets: Offset values for affinity computation used as target.
-        boundaries: Whether to compute boundaries as the target.
-        binary: Whether to use a binary segmentation target.
-        kwargs: Additional keyword arguments for `torch_em.default_segmentation_dataset`.
```

Comment:
This looks wrong.  These parameters are still there, so we shouldn't remove the doc.

Reply:
Ah yeah my bad. It should be restored now. Thanks for spotting!